content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -107,44 +109,52 @@ If the operations succeed, the device plugin returns an `AllocateResponse` that
 runtime configurations for accessing the allocated devices. The kubelet passes this information
 to the container runtime.
 
+### Handling kubelet restarts
+
 A device plugin is expected to detect kubelet restarts and re-register itself with the new
 kubelet instance. In the current implementation, a new kubelet instance deletes all the existing Unix sockets
 under `/var/lib/kubelet/device-plugins` when it starts. A device plugin can monitor the deletion
 of its Unix socket and re-register itself upon such an event.
 
 ## Device plugin deployment
 
-A device plugin can be deployed manually or as a DaemonSet. Being deployed as a DaemonSet has
-the benefit that Kubernetes can restart the device plugin if it fails.
+A device plugin can be deployed manually or as a DaemonSet. If you deploy a device
+plugin as a DaemonSet, you can benefit from having Kubernetes take care of restarting
+the device plugin if it fails.
 Otherwise, an extra mechanism is needed to recover from device plugin failures.
+
 The canonical directory `/var/lib/kubelet/device-plugins` requires privileged access,
 so a device plugin must run in a privileged security context.
 If a device plugin is running as a DaemonSet, `/var/lib/kubelet/device-plugins`
-must be mounted as a
-[Volume](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volume-v1-core)
+must be mounted as a {{< glossary_tooltip term_id="volume" >}}
 in the plugin's
 [PodSpec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core).
 
-Kubernetes device plugin support is in beta. As development continues, its API version can
-change. We recommend that device plugin developers do the following:
+## API compatibility
+
+Kubernetes device plugin support is in beta. The API may change before stabilization,
+in incompatible ways. As a project, Kubernetes recommends that device plugin developers:
```

Comment:
Sigh ... "in incompatible ways"? That is not what "beta" stage designed for. Anyway, the developers really wanted to be in Beta, for whatever reasons.

Reply:
Is it OK to leave in the sentiment & revised wording?

Reply:
I share @tengqm's grumbling, but yes, I think it's OK to leave in place while flagging for a larger conversation about feature states.